app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+  after_create :increase_counter
+
+  private
+
+  def increase_counter
```

Comment:
### Kudos for making these methods private 👏

Reply:
Thank you 😃